filter_plugins/oo_filters.py

```diff
@@ -601,36 +664,38 @@ def oo_persistent_volumes(hostvars, groups, persistent_volumes=None):
 
         if persistent_volumes == None:
             persistent_volumes = []
-        for component in hostvars['openshift']['hosted']:
-            kind = hostvars['openshift']['hosted'][component]['storage']['kind']
-            create_pv = hostvars['openshift']['hosted'][component]['storage']['create_pv']
-            if kind != None and create_pv:
-                if kind == 'nfs':
-                    host = hostvars['openshift']['hosted'][component]['storage']['host']
-                    if host == None:
-                        if len(groups['oo_nfs_to_config']) > 0:
-                            host = groups['oo_nfs_to_config'][0]
+        if 'hosted' in hostvars['openshift']:
```

Comment:
We may want to see if we can clean this up at some point in the future, this is going to get big and hairy as we continue to add storage types.